layouts/project/single.html

```diff
@@ -2,29 +2,25 @@
 	<div class="margin-bottom-30">
 		{{ .Content }}
 	</div>
-	{{ if gt (len .Params.eclipse_projects) 0 }}
-	<h2 class="margin-bottom-40">Eclipse projects</h2>
-	<div class="row">
-	{{ range .Params.eclipse_projects }}
-		{{range first 1 (where $.Site.Data.eclipsefdn_projects "project_id" .) }}
-		<div class="col-sm-12 margin-bottom-30 match-height-item-by-row flex-column project-item">
-		  <div class="padding-bottom-20 text-center background-white flex-grow">
-		    <div >
-		      <h3>
-		        <a href="{{ .url }}">
-			      <img class="img img-responsive margin-auto padding-bottom-20 " src="{{ .logo }}" title="{{ .name }} logo"/>
-				  {{ .name }}
-			    </a>
-			  </h3>
+	{{ if .Params.referenced_eclipse_projects }}
+		{{ if gt (len .Params.referenced_eclipse_projects) 0 }}
+			<h2 class="margin-top-30 margin-bottom-20">Referenced Eclipse projects</h2>
+			<div class="row">
+			{{ range .Params.referenced_eclipse_projects }}
+				{{range first 1 (where $.Site.Data.eclipsefdn_projects "project_id" .) }}
+				<div class="col-md-4 background-white margin-bottom-20 match-height-item vertical-align margin-right-20">
```

Comment:
This should be col-sm-4 at the most. Md means that up to ~900px there is one tile per row which doesn't look quite right

Reply:
But it is a little bit weird that between 700-900px, the referenced projects logos are small, but the logos in sidebar are big. Do you think I should adjust them or just leave it like this? 🤔

![image](https://user-images.githubusercontent.com/39588094/95461744-99576600-0944-11eb-9e37-36b6ee82a472.png)

Reply:
You could up it to col-sm-6 if you like. The point was more that we needed the small columns covered as well as medium at least.